Sources/CNIOAtomics/include/CNIOFastAtomics.h

```diff
@@ -0,0 +1,280 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
```

Comment:
Mind setting this date to `2019` instead?

Reply:
I updated CNIOAtomics.h since this one is getting removed.